app/javascript/components/time_table/timesheet/history/entry_history.js

```diff
@@ -334,6 +348,27 @@ class EntryHistory extends React.Component {
     }
   }
 
+  onTagFilter(e) {
+    e.preventDefault();
+    const { from, to } = this.state;
+    const sTag = e.target.attributes.getNamedItem('data-tag').value;
+    if (sTag) {
+      this.setState({
+        selectedTag: _.find(this.props.tags, tag => tag.key === sTag).value,
```

Comment:
I think that Array's find would work as well

Reply:
We don't need filtering here in entry_history so this line won't exist anyways